docs/references/backend/types/backend-session.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: The Backend Session object
+description: The Backend Session object holds information about a Session of your application. However, the Backend Session object is different from the Session object in that it is used in the Backend API.
+---
+
+# The Backend `Session` object
+
+The Backend `Session` object is similar to the [`Session`](/docs/references/javascript/session) object as it is an abstraction over an HTTP session and models the period of information exchange between a user and the server. However, the Backend `Session` object is different from the `Session` object in that it is used in the [Backend API](/docs/reference/backend-api/tag/Sessions#operation/GetSessionList) and is not directly accessible from the Frontend API.
```

Comment:
Do you mean "Backend Session object is different from the Session object in that it is used in the [Frontend API](https://clerk.com/docs/pr/1231/reference/frontend-api/tag/Sessions#operation/GetSessionList) and is not directly accessible from the Frontend API."

Reply:
No, because the Backend `Session` object is used in the Backend API, while the `Session` object is used in the Frontend.

Reply:
I could change the wording to be less confusing,

```suggestion
The Backend `Session` object is similar to the [`Session`](/docs/references/javascript/session) object as it is an abstraction over an HTTP session and models the period of information exchange between a user and the server. However, the Backend `Session` object is different from the `Session` object as it is used in the [Backend API](/docs/reference/backend-api/tag/Sessions#operation/GetSessionList) and is not directly accessible from the Frontend API.
```

Reply:
Used this wording ^^